scm/driver/github/testdata/webhooks/pipeline_hook.json.golden

```diff
@@ -0,0 +1,50 @@
+{
+  "Repo": {
+    "ID": "335170389",
+    "Namespace": "shivamnegi94",
+    "Name": "testrepo",
+    "Clone": "https://github.com/shivamnegi94/testrepo.git",
+    "CloneSSH": "git@github.com:shivamnegi94/testrepo.git",
+    "Link": "https://github.com/shivamnegi94/testrepo",
+    "Branch": "main",
+    "Created": "2021-02-02T04:41:22Z",
+    "Updated": "2025-03-24T08:01:37Z",
+    "Private": false,
+    "Visibility": 1,
+    "Perm": {}
+  },
+  "Commit": {
+    "Sha": "f4229db4b70a8dadeb687f5ccce5fd327c04461a",
+    "Message": "Update NewSre_v1.yaml",
+    "Author": {
+      "Name": "Shivam Negi",
+      "Email": "shivam.negi@harness.io"
+    },
+    "Committer": {
+      "Name": "GitHub",
+      "Email": "noreply@github.com"
+    }
+  },
+  "Execution": {
+    "Number": 14030131450,
```

Comment:
execution number should map to `"run_number": 2`

Reply:
Updated the field to run_number